packages/lab/src/dropdown/DropdownButton.tsx

```diff
@@ -90,7 +90,7 @@ export const DropdownButton = forwardRef(function DropdownButton(
       )}
       // We don't want the 'button' tag to be shown in the DOM to trigger some accessibility testing
```

Comment:
This comment isn't valid now. We could either remove it or put it above the return